### PR TITLE
A few things...

### PR DESCRIPTION
**Pull request by g-sam**
Check out commit msgs and notes on them

### CODE REVIEW

File: misc/basic_keys.py [hunk 1]
```diff
@@ -1,89 +1,76 @@
...
+simple_keys = normalise_keys(
+    {
+        "( backspace | junk )": "backspace",
+        "(delete | forward delete | scrap | spunk)": "delete",
+        # voicecode alternates
+        "(space | skoosh)": "space",
+        "(tab | tarp)": "tab",
+        "(go left | crimp | lloyd)": "left",
+        "(go right | chris)": "right",
+        "(go up | jeep)": "up",
```
Comment (dwiel) [inline misc/basic_keys.py]: `go up up up left` no longer works

Comment (g-sam) [reply misc/basic_keys.py]: good point. will fix

**Comment by g-sam** [comment]
that should be fixed now. 

were you able to get the pre commit hook working?  there were some limiting errors  from previous commits I had to fix before I could push.  best thing would be to both use the pre commit hook and put a check on master to prevent merges that don't pass CI.

**Comment by dwiel** [comment]
Dog, forgot about pre commit hooks, and I agree with CI blocking. Thanks for fixing my commits

**Pull request merged**